client/lib/signup/step-actions/index.js

```diff
@@ -181,10 +179,6 @@ function getNewSiteParams( {
 			site_creation_flow: flowToCheck,
 			timezone_string: guessTimezone(),
 			wpcom_public_coming_soon: 1,
-			difm_lite_site_options: {
```

Comment:
These have been removed since these options are now passed in the `extra` field of the cart item.